api/tests/local_providers/cds_stocks_test.py

```diff
@@ -0,0 +1,131 @@
+from unittest.mock import patch
+
+import pytest
+
+from pcapi.core.booking_providers.models import Movie
+from pcapi.core.categories import subcategories
+from pcapi.core.offers.models import Offer
+from pcapi.core.providers.factories import VenueProviderFactory
+from pcapi.core.providers.models import Provider
+from pcapi.local_providers.cinema_providers.cds.cds_stocks import CDSStocks
+from pcapi.models.product import Product
+
+from tests.conftest import clean_database
+
+
+@pytest.mark.usefixtures("db_session")
+class CDSStocksTest:
+    @clean_database
+    @patch("pcapi.core.booking_providers.cds.client.CineDigitalServiceAPI.get_venue_movies")
+    def should_get_venue_movies(self, mock_get_venue_movies):
+
+        # Given
+        cds_provider = Provider.query.filter(Provider.localClass == "CDSStocks").first()
```

Comment:
```suggestion
        cds_provider = Provider.query.filter(Provider.localClass == "CDSStocks").one()
```
`first()` est piégeux, il ne devrait pas y avoir ici moins ou plus que 1 seul Provider qui correspond à la requête.